src/Collection/Hero.php

```diff
@@ -293,6 +327,7 @@ public static function format_background( $value, $post_id, $field ) {
 
 			if ( 'video' === $item['type'] ) {
 				$new_value[ $index ]['video'] = Utils::get_video_embed_url( $item['video'] );
```

Comment:
I left for backward compatibility - so if someone were already using this feature it would still work. I guess there is a risk that it will do something funny if you had a video whose file name matched the rules. Do you think the safer option is to just bump up a major version and remove this?